docs/process/rust_versions.md

```diff
@@ -3,7 +3,7 @@
 
 ## For ICU4X users
 
-Each ICU4X release and its patches guarantee a minimum supported rust version (MSRV) of four Rust versions before the release, and attempt an MSRV of six Rust versions. For example, if today's current Rust stable version is 1.70, and we release ICU4X 1.3, then we guarantee that ICU4X 1.3 will have an MSRV of 1.66 or earlier. We will *attempt* an MSRV of 1.64.
+Each ICU4X release and its patches guarantee a minimum supported rust version (MSRV) of four Rust versions before the release, and attempt an MSRV of six Rust versions. For example, if today's current Rust stable version is 1.70, and we release ICU4X x.y.0, then we guarantee that ICU4X x.y.z will have an MSRV of 1.66 or earlier. We will *attempt* an MSRV of 1.64.
```

Comment:
observation: this phrasing allows a bump from n-6 to n-4 in a patch release

Reply:
This is a confusingly worded paragraph and I didn't want to re-write the whole thing since it was the result of a careful hour-long discussion. I don't know if I have the same interpretation, _but_ if changing from n-6 to n-4 fixes a soundness issue, then that might be acceptable from my point of view. However, we shouldn't change from n-6 to n-4 for no reason, and I don't think this phrasing allows for that.